app/src/js/components/Executions/execution-status.js

```diff
@@ -1,290 +1,109 @@
-                >
-                  Show Output
-                </button>
-                <DefaultModal
-                  showModal={showOutputModal}
-                  title={
-                    <>
-                      <span>Execution Output</span>
-                      <a
-                        className="button button--small button--download button--green form-group__element--right"
-                        id="download_link"
-                        download="output.json"
-                        href={downloadUrl}
-                      >
-                        Download File
-                      </a>
-                    </>
-                  }
-                  onCloseModal={() => this.closeModal('output')}
-                  hasConfirmButton={false}
-                  cancelButtonClass="button--close"
-                  cancelButtonText="Close"
-                  className="execution__modal"
-                >
-                  <pre>{parseJson(d)}</pre>
-                </DefaultModal>
-              </>
-            );
-          }
-          return 'N/A';
-        },
-      },
-      {
-        label: 'Logs',
-        property: 'executionArn',
-        accessor: (d) => {
-          const kibanaLink = kibanaExecutionLink(cumulusInstance, d);
-          const className =
-            'button button--small button__goto button__arrow button__animation button__arrow--white';
-          if (kibanaLink && kibanaLink.length) {
-            return (
-              <a href={kibanaLink} target="_blank" className={className}>
-                View Logs in Kibana
-              </a>
-            );
-          }
-          return (
-            <Link
-              to={(location) => ({
-                pathname: `/executions/execution/${d}/logs`,
-                search: getPersistentQueryParams(location),
-              })}
-              title={`${d}/logs`}
-              className={className}
-            >
-              View Execution Logs
-            </Link>
-          );
-        },
-      },
-    ];
-
-    return (
-      <div className="page__component">
-        <Helmet>
-          <title> Execution Status </title>
-        </Helmet>
-        <section className="page__section page__section__header-wrapper">
-          <h1 className="heading--large heading--shared-content with-description width--three-quarters">
-            Execution: {name}
-          </h1>
-
-          {error && <ErrorReport report={error} />}
-        </section>
-
-        {/* stateMachine's definition and executionHistory's event statuses are needed to draw the graph */}
-        {stateMachine && executionHistory && (
-          <section className="page__section">
-            <div className="heading__wrapper--border">
-              <h2 className="heading--medium with-description">Visual</h2>
-            </div>
-            <ExecutionStatusGraph executionStatus={executionStatus} />
-          </section>
-        )}
-
```

Comment:
Primary change here is moving metaAccessors into the table-config file and simplifying the modal toggling.